scripts/install_tethys.bat

```diff
@@ -0,0 +1,61 @@
+# WARNING this is completely untested!!!!
```

Comment:
I should also say that I won't have time to work on this anymore for a while, so I was hoping you could pick it up from here.